satpy/etc/composites/atms.yaml

```diff
@@ -14,5 +14,5 @@ composites:
     prerequisites:
     - name: '16'
     - name: '17'
-    - name: '22'
+    - name: '18'
```

Comment:
This isn't related to AWS, so shouldn't be part of this PR. Right?

Reply:
Yeah, you are right. I am revising the composite recipes now with AWS...